anoma/namada-interface
# feat/1055 - Faucet settings

Resolves #1055 

- Added settings modal to configure Faucet API url with basic persistence (localStorage)
- Restructured a bit
- Faucet Limit is now determined by settings (from API)
- Updated components so that forms submit on Enter key

### Testing

- Build and load the faucet app as per usual, in the top right, click the gear icon
- Enter a new valid URL, then submit
- If this new URL cannot load settings, you will see an error. Setting this to a valid API url will reload the settings and clear the error
- Reload page to ensure this was persisted, or check the local storage key for `baseUrl`
